Neos.Flow/Classes/I18n/Xliff/XliffParser.php

```diff
@@ -28,67 +26,26 @@
  * @throws Exception\InvalidXliffDataException
  * @see http://docs.oasis-open.org/xliff/v1.2/os/xliff-core.html [1]
  * @see http://docs.oasis-open.org/xliff/v1.2/xliff-profile-po/xliff-profile-po-1.2-cd02.html#s.detailed_mapping.tu [2]
+ * @deprecated use \Neos\Flow\I18n\Xliff\V12\XliffParser instead. Will be removed in Flow 5.0
```

Comment:
Can we already remove this in the next major, when we deprecate something in a minor release? Or do we have to wait for 6.0 to finally remove it?